content/_index.md

```diff
@@ -25,23 +25,41 @@ OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI
 
 ## Install
 
+![OSM Install Demo](https://github.com/openservicemesh/osm/raw/release-v0.8/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
+
 ### Prerequisites
-- Kubernetes cluster running Kubernetes v1.18.0 or greater
-- kubectl current context is configured for the target cluster install
-  - ```kubectl config current-context```
+- Kubernetes cluster running Kubernetes v1.18.0 or greater.
+- `kubectl` installed and configured to connect to your Kubernetes cluster.
+  - To display the cluster `kubectl` is configured to connect to, use `kubectl config current-context`
 
-### Get the OSM CLI
+### Install the OSM CLI
 
-The simplest way of installing Open Service Mesh on a Kubernetes cluster is by using the `osm` CLI.
+The `osm` CLI is the simplest way to install OSM on your Kubernetes cluster. To install the `osm` CLI, download and unpack the `osm` binary from the [Releases page](https://github.com/openservicemesh/osm/releases) and add it to `$PATH`. For more details on installing the `osm` CLI, see [Installation](https://docs.openservicemesh.io/docs/install/).
 
-Download the `osm` binary from the [Releases page](https://github.com/openservicemesh/osm/releases). Unpack the `osm` binary and add it to `$PATH` to get started.
-```shell
-sudo mv ./osm /usr/local/bin/osm
-```
+### Install OSM on your Kubernetes cluster
+
+To install OSM on your cluster using the `osm` CLI, use the `install` command. For example:
 
-### Install OSM
 ```shell
 $ osm install
 ```
 
-![OSM Install Demo](https://github.com/openservicemesh/osm/raw/release-v0.8/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
+Alternatively, you can install OSM on your cluster using a Helm chart. You can also use the `osm` CLI to install OSM on OpenShift. For more details on installation options, see [Install OSM](https://docs.openservicemesh.io/docs/install/#install-osm).
+
+## Demos
```

Comment:
This is great content. I wonder if we should add a demos.md page in https://github.com/openservicemesh/osm-docs/tree/main/content/docs so that we don't end up with an unmaintainably long list on the front page? If we want to do some kind of "featured" content, sure, but the permanent home for this info does not seem like it should be "the front page"?

Reply:
moved to a new page